src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs

```diff
@@ -198,14 +198,15 @@ private enum ResponseStatus
             return authParams;
         }
 
-        private static MsalException DecorateExceptionWithRuntimeErrorProperties(MsalException exception, Error runtimeError)
-        {
+        private static MsalException DecorateExceptionWithRuntimeErrorProperties(MsalException exception, AuthResult wamAuthResult)
```

Comment:
Should the naming be more generic, like `brokerAuthResult`?

Reply:
this is internal .... but ok.